### PR TITLE
Fix patch 2.01 + script compilation

### PR DESCRIPTION
**Pull request by psiberx**
Fixes script compilation when REDmod is used.

### CODE REVIEW

**Review by alghaibb — APPROVED** [review]
Wouldn't mind trying this out to see if it works

**Comment by Giotje** [comment]
@psiberx do you have the updated files somehwere? i can't seem to build it for some reason

**Comment by extract** [comment]
> do you have the updated files somehwere? i can't seem to build it for some reason

If you need the build files they are available in github actions

https://github.com/WopsS/RED4ext/actions

But be aware that it's not working yet.

**Comment by Cybersific** [comment]
Following progress, good work guys

**Review by HGinnerup — CHANGES_REQUESTED** [review]
Build fails with `'RED4EXT_RUNTIME_2_01': undeclared identifier`

Replacing `(aVersion == RED4EXT_RUNTIME_2_01)` with `true`, still gets blocked by `Image::IsSupported()` returning false.

Making `Image::IsSupported()` always return true, makes the game attempt to load, but seemingly crashes when it gets to the first mod, though it may just be that ArchiveXL also needs to update something.

```
[2023-10-05 20:36:21.983] [RED4ext] [info] RED4ext (v0.0.0+master.d4b3262.20231005T180444Z) is initializing...
[2023-10-05 20:36:21.983] [RED4ext] [info] Game patch: 2.01
[2023-10-05 20:36:21.983] [RED4ext] [info] Product version: 2.01
[2023-10-05 20:36:21.983] [RED4ext] [info] File version: 3.0.74.63017
[2023-10-05 20:36:22.067] [RED4ext] [info] RED4ext has been successfully initialized
[2023-10-05 20:36:22.146] [RED4ext] [info] RED4ext is starting up...
[2023-10-05 20:36:22.146] [RED4ext] [info] Loading plugins...
[2023-10-05 20:36:22.147] [RED4ext] [info] Loading plugin from '[...]\Cyberpunk 2077\red4ext\plugins\ArchiveXL\ArchiveXL.dll'...
```

File: src/dll/Utils.cpp [hunk 1]
```diff
@@ -259,6 +259,10 @@ std::wstring Utils::FileVerToPatch(const RED4ext::FileVer& aVersion)
     {
         return L"2.00";
     }
+    else if (aVersion == RED4EXT_RUNTIME_2_01)
```
[the remark this answers is not in the document]
Comment (extract) [reply src/dll/Utils.cpp:262]: you need the submodule updated with https://github.com/WopsS/RED4ext.SDK/pull/92 ... just wait.

[the remark this answers is not in the document]
Comment (HGinnerup) [reply src/dll/Utils.cpp:262]: Sorry, you're right. Updated the submodule as well, and it seems to work.

**Review by DuskDweller — APPROVED** [review]

**Review by WopsS — APPROVED** [review]
Thanks!

**Pull request merged**